INSTALL.md

```diff
@@ -36,19 +38,79 @@ yarn add @dash0/sdk-web
      serviceName: "my-website",
      endpoint: {
        // Replace this with the endpoint url identified during preparation
-       url: REPLACE THIS,
+       url: "REPLACE THIS",
        // Replace this with your auth token you created earlier
        // Ideally, you will inject the value at build time in order not commit the token to git,
        // even if its effectively public in the HTML you ship to the end user's browser
-       authToken: REPLACE THIS
+       authToken: "REPLACE THIS",
      },
    });
    ```
 
+### Adding the Dash0 Web SDK via script tags
+
+The Dash0 Web SDK can also injected via script tags, which is useful for website not using on module builds.
+To add the Dash0 Web SDK to the HTML of your website, add the snippet below and adjust the configuration as needed.
+
+```html
+<script>
+  (function (d, a, s, h, z, e, r, o) {
+    d[a] ||
+      ((z = d[a] =
+        function () {
+          h.push(arguments);
+        }),
+      (z._t = new Date()),
+      (z._v = 1),
+      (h = z._q = []));
+  })(window, "dash0");
+  dash0("init", {
+    serviceName: "my-website",
+    endpoint: {
+      // Replace this with the endpoint url identified during preparation
+      url: "REPLACE THIS",
+      // Replace this with your auth token you created earlier
+      // Ideally, you will inject the value at build time in order not commit the token to git,
+      // even if its effectively public in the HTML you ship to the end user's browser
+      authToken: "REPLACE THIS",
+    },
+  });
+</script>
+<!--Latest version-->
+<script defer crossorigin="anonymous" src="https://unpkg.com/@dash0/sdk-web/dist/dash0.iife.js"></script>
+<!--Or pin a specific version-->
+<script defer crossorigin="anonymous" src="https://unpkg.com/@dash0/sdk-web@0.18.1/dist/dash0.iife.js"></script>
+```
+
+You can choose to always load the latest version of the Dash0 Web SDK or pin the script to a specific version (see example below).
+Loading a specific version of the Dash0 Web SDK usually improves loading performance of the script.
+
+#### Api usage
+
+Please note that the API for the IIFE build of the Dash0 Web SDK is slightly different from the module build.
+All APIs must be called via a global `dash0` function. The following call `addSignalAttribute("the_answer", 42)` for example
+would called like this for the IIFE build: `dash0("addSignalAttribute", "the_answer", 42)`.
```

Comment:
For example, the following call `addSignalAttribute("the_answer", 42)` would be called like this for the IIFE build: `dash0("addSignalAttribute", "the_answer", 42)`.